stacks/production.tfvars

```diff
@@ -2,6 +2,7 @@ remote_state_bucket = "prometheus-production"
 stack_name = "production"
 dev_environment = "false"
 prometheus_subdomain = "monitoring"
+targets_s3_bucket = "gds-prometheus-targets"
```

Comment:
Why does this one not have the `prod` suffix? ;)

Reply:
You're meant to be on holiday! Also because it's the default behaviour, prod's so important it doesn't need a suffix

Reply:
although obviously if this is confusing can change :)